webrtc-stats.html

```diff
@@ -1554,9 +1554,17 @@ <h2>
               </dt>
               <dd>
                 <p>
+                  The purpose of the jitter buffer is to recombine RTP packets into frames (in the case of video)
+                  and have smooth playout. The model described here assumes that the samples or frames are 
+                  still compressed and have not yet been decoded.
```

Comment:
Is it correct to add: "...not yet ben decoded, but a valid implementation could decode the packets prior to insertion into the jitter buffer." ?

Reply:
Typically, Jitter buffer makes sure it hands the full frame for decoding and also approximates the time to playout. So major part of the buffering is done here and not downstream in the playout buffer, which is a small circular buffer (keeps a frame or few)

Reply:
If there is down stream delays or buffer. we need a different buffer metric, for example a playout buffer metric.